File: homework3-4-master/app/src/main/java/com/example/home_work_3_4/MainActivity.java
package com.example.home_work_3_4;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.example.home_work_3_4.databinding.ActivityMainBinding;


public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_continents, new ContinentsFragment()).addToBackStack(null).commit();
    }
}

File: homework3-5-master/app/src/main/java/com/example/homework3_5/MainActivity.kt
package com.example.homework3_5

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.homework3_5.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        supportFragmentManager.beginTransaction().add(R.id.fragment_container, FirstFragment()).commit()
    }
}